cmd/juju/commands/main.go

```diff
@@ -146,6 +146,9 @@ func (m jujuMain) Run(args []string) int {
 
 	if err = juju.InitJujuXDGDataHome(); err != nil {
 		cmd.WriteError(ctx.Stderr, err)
+		if errors.Is(err, errors.NotFound) {
+			ctx.Warningf("Installing juju from a snap, the .local/share directory should be manually created as it is strictly confined.")
```

Comment:
A sentence like this, where you state 2 things at the front of the sentence, and then refer to "it" at the back, makes it ambiguous.

It should be something like:
"Installing Juju in a strictly confined Snap. To ensure correct operation, create the .local/share directory manually."